evm/vm/forks/spurious_dragon/__init__.py

```diff
@@ -52,17 +53,17 @@ def _apply_spurious_dragon_create_message(vm, message):
     else:
         contract_code = computation.output
 
-        if contract_code and len(contract_code) >= constants.EIP170_CODE_SIZE_LIMIT:
-            computation._error = OutOfGas(
+        if contract_code and len(contract_code) >= EIP170_CODE_SIZE_LIMIT:
+            computation.error = OutOfGas(
```

Comment:
This change should be reverted.

Reply:
This definitely fixed it! Glad you noticed it--I must've done this somehow by mistake when changing the constant reference on the line above.